src/components/app/MenuButtons.js

```diff
@@ -178,15 +178,15 @@ function _formatVersionNumber(version?: string): string | null {
   return null;
 }
 
-function _formatLabel(meta: ProfileMeta): string | null {
+function _formatLabel(meta: ProfileMeta): string {
   const product = meta.product || '';
   const version = _formatVersionNumber(meta.misc) || '';
   const os = meta.oscpu || '';
 
   const labelTitle = product + ' (' + version + ') ' + os;
 
   if (labelTitle.length < 5) {
-    return null;
+    return '';
```

Comment:
It's probably not important but is there a reason to change that here?

Reply:
It is mandatory because we are assigning `label={_formatLabel(meta)}` in `ButtonWithPanel` component which is of type string and it cannot be null.